app/editor/src/store/hooks/lookup/utils/fetchIfNoneMatch.ts

```diff
@@ -0,0 +1,46 @@
+import { AxiosError, AxiosResponse } from 'axios';
+import { ICacheModel } from 'hooks/api-editor';
+
+import { addOrUpdate, getFromLocalStorage, initFromLocalStorage } from '.';
+
+export const fetchIfNoneMatch = async <T>(
```

Comment:
I need to document this.  This function does a lot of magic for the hooks, redux store, error handling, and ETags.